internal/providers/terraform/aws/ec2_client_vpn_endpoint.go

```diff
@@ -0,0 +1,38 @@
+package aws
+
+import (
+	"github.com/infracost/infracost/internal/schema"
+
+	"github.com/shopspring/decimal"
+)
+
+func GetEC2ClientVPNEndpointRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "aws_ec2_client_vpn_endpoint",
+		RFunc: NewEC2ClientVPNEndpoint,
+	}
+}
+
+func NewEC2ClientVPNEndpoint(d *schema.ResourceData, u *schema.ResourceData) *schema.Resource {
+	region := d.Get("region").String()
+
+	return &schema.Resource{
+		Name: d.Address,
+		CostComponents: []*schema.CostComponent{
+			{
+				Name:           "Client VPN connection",
```

Comment:
```suggestion
				Name:           "Connection",
```

Given that the resource is already has `client_vpn`, and it only has 1 cost component, I think Connection is simpler to understand?